gadgets/profile_tcprtt/README.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: profile_tcprtt
+sidebar_position: 30
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# profile_tcprtt
+
+The `profile_tcprtt` gadget generates a histogram distribution of the TCP
+connections' Round-Trip Time (RTT). The RTT values used to create the histogram
+are collected from [the smoothed
+RTT](https://github.com/torvalds/linux/blob/v5.11/include/linux/tcp.h#L258)
+information already provided by the Linux kernel for the TCP sockets.
+
+The histogram considers only the TCP connections that have been already
+established, so it does not take into account the connection phase (3-way TCP
+Handshake). If it is what you are looking for, please check the latency
+information the `trace_tcpconnect` gadget provides. See further information
+[here](../trace_tcpconnect.mdx).
```

Comment:
```suggestion
in the [trace_tcpconnect documentation](trace_tcpconnect.mdx).
```

The link relative path was wrong because of the way symlinks are done in `docs/gadgets/`